File: app/src/main/java/com/xjm/xxd/dribbird/splash/ISplashActivityPresenter.java
package com.xjm.xxd.dribbird.splash;

import com.xjm.xxd.dribbird.base.IPresenter;

/**
 * Created by queda on 2016/12/3.
 */

public interface ISplashActivityPresenter extends IPresenter<SplashActivityView> {

    // 检测是否存在已登陆的账户
    void checkAccount();

}

File: app/src/main/java/com/xjm/xxd/dribbird/splash/ISplashActivityPresenter.kt
package com.xjm.xxd.dribbird.splash

import com.xjm.xxd.framework.base.IPresenter

/**
 * Created by queda on 2016/12/3.
 */

interface ISplashActivityPresenter : IPresenter<SplashActivityView> {

    // 检测是否存在已登陆的账户
    fun checkAccount()

}
